docs/overview.rst

```diff
@@ -0,0 +1,29 @@
+========
+Overview
+========
+
+Ctwrap is a light-weight batch simulation wrapper written in Python. The wrapper consists of two modules:
+the ``parser`` and the ``simulation`` modules. The ``parser`` module is used to parsed the input file and also
```

Comment:
Could you rephrase this in terms of objects/classes rather than modules? I believe the important things here are `Parser`, `Simulation` and `SimulationHandler`. You mention this further down, but it may be better to skip to the important part immediately.